packages/web-pkg/src/components/BatchActions.vue

```diff
@@ -46,6 +46,17 @@ export default defineComponent({
 
 <style lang="scss">
 #oc-appbar-batch-actions {
+  .action-menu-item {
+    padding-left: var(--oc-space-small) !important;
+    padding-right: var(--oc-space-small) !important;
+    gap: var(--oc-space-xsmall) !important;
+  }
+  .action-menu-item:hover:not([disabled]),
+  .action-menu-item:focus:not([disabled]) {
+    background-color: var(--oc-color-background-highlight);
+    border-color: var(--oc-color-background-highlight);
```

Comment:
Talked with @tbsbdr : Please use `var(--oc-color-background-hover)` here. In a next step we will make the hover color the current highlight color so this will be blue eventually.

Reply:
done